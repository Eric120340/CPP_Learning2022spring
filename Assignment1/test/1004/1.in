5 5 3
5 46 43 17 34 
20 8 16 4 21 
4 31 49 17 20 
47 9 39 3 50 
39 35 3 22 15 
