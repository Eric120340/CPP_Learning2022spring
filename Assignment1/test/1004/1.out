312419 427488 459346 212430 395732 
177822 209922 239459 105858 213490 
268338 366813 402392 185457 336389 
392824 443653 522867 224060 469343 
276722 373073 375009 189943 326504 
